src/zope/testrunner/options.py

```diff
@@ -418,14 +418,14 @@ def _regex_search(s):
 
 setup.add_argument(
     '--package-path', action="append", dest='package_path', nargs=2,
+    metavar="ARG",
     help="""\
 Specify a path to be searched for tests, but not added to the Python
 search path.  Also specify a package for files found in this path.
 This is used to deal with directories that are stitched into packages
 that are not otherwise searched for tests.
 
-This option takes 2 arguments.  The first is a path name. The second is
-the package name.
+This option takes 2 arguments specifying the path and the package.
```

Comment:
BTW do you have any idea what this option actually does?  Because I don't, and the text isn't helping.  Maybe an example would help?